src/components/ManagementDataSection/ManagementDataSection.jsx

```diff
@@ -77,11 +72,7 @@ const ManagementDataSection = ({
 
   const displaySortByButton = () => {
     return (
-      <DropdownButton
-        className={styles['dropdown-button']}
-        id="dropdown-basic-button"
-        title="Sort By"
-      >
+      <DropdownButton id="dropdown-basic-button" title="Sort By">
```

Comment:
There's also several changes to this file. I'm not sure if they're intentional but I don't think they should be here.

Reply:
Sorry im not sure why there were so many changes